.mergify.yml

```diff
@@ -0,0 +1,27 @@
+pull_request_rules:
+  - name: assign and label scala-steward's PRs
+    conditions:
+      - author=scala-steward
+    actions:
+      assign:
+        users: [sideeffffect]
+      label:
+        add: [dependency-update]
+  - name: merge Scala Steward's PRs
+    conditions:
+      - base=master
+      - author=scala-steward
+      - "body~=(labels: library-update, semver-minor)|(labels: library-update, semver-patch)|(labels: sbt-plugin-update, semver-minor)|(labels: sbt-plugin-update, semver-patch)|(labels: scalafix-rule-update, semver-minor)|(labels: scalafix-rule-update, semver-patch)|(labels: test-library-update, semver-minor)|(labels: test-library-update, semver-patch)"
+      - "status-success=license/cla"
+      - "status-success=lint"
+      - "status-success=test (adopt@1.8, 2.11.12, JVM)"
+      - "status-success=test (adopt@1.8, 2.12.13, JVM)"
+      - "status-success=test (adopt@1.8, 2.13.4, JVM)"
+      - "status-success=test (adopt@1.8, 3.0.0-M3, JVM)"
+      - "status-success=test (adopt@1.11, 2.11.12, JVM)"
+      - "status-success=test (adopt@1.11, 2.12.13, JVM)"
+      - "status-success=test (adopt@1.11, 2.13.4, JVM)"
+      - "status-success=test (adopt@1.11, 3.0.0-M3, JVM)"
```

Comment:
I'd drop the platform part as suggested above. In the meantime, I'll grant Mergify permissions for this repo.